### PR TITLE
`hotshot/rc-0.5.81`

### PR DESCRIPTION
**Pull request by rob-maron**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -6,18 +6,14 @@ edition = "2021"
 
 [dependencies]
 async-broadcast = "0.7"
-async-compatibility-layer = { version = "1.1", default-features = false, features = [
-    "logging-utils",
-] }
 async-lock = "2.8"
```
Comment (imabdulbasit) [inline Cargo.toml:9]: can we also update this async--lock to "3" here so that we can compile sequencer without lockfile

File: Cargo.toml [hunk 2]
```diff
@@ -26,8 +22,15 @@ tide-disco = "0.9"
 toml = "0.8"
 tracing = "0.1"
 vbs = "0.1.4"
+tokio = { version = "1", default-features = false, features = [
```
Comment (imabdulbasit) [inline Cargo.toml:25]: also what is the reason to just use tokio now and remove async-std?

File: Cargo.toml (hunk 1 above)
Comment (rob-maron) [reply Cargo.toml:9]: Sure

File: Cargo.toml (hunk 2 above)
Comment (rob-maron) [reply Cargo.toml:25]: - We only have to support a single executor in HotShot
- It lets us use async primitives unsupported by the compatibility layer
- Development would no longer involve manually setting RUSTFLAGS everywhere
- It lets us migrate off of tide to a more modern Tokio-based framework (we can technically do this now if we rock `tokio1`, but I think it's best to have everything on a single executor)

In addition to the above, Tokio is better maintained

**Comment by sveitser** [comment]
Since https://github.com/EspressoSystems/hotshot-events-service/pull/123 we are on 0.5.82 of hotshot, so seems reasonable to close this.

**Pull request closed without merging**